### ISSUE TITLE
Use GoogleMap for Geo Autocomplete 

### ISSUE THREAD

**Issue by pH-7** [pH7Software/pH7-Social-Dating-CMS#78]
https://developers.google.com/maps/documentation/javascript/examples/places-autocomplete-addressform

Ref: https://github.com/pH7Software/pH7-Social-Dating-CMS/commit/2d008baec45a082bfbf9a471cf0e62f430774aa0

**Comment by pH-7** [comment]
Or even https://github.com/xdan/autocomplete
Demo http://xdsoft.net/jqplugins/autocomplete

**Comment by rondlite** [comment]
This is in the script already, just needed a tweak...
When I type a city it looks up the cities and suggests them.... is this different?

**Comment by Undefined-Variables** [comment]
Is this part of the latest release? I have changed the list of US states for UK counties, not done much for last few weeks though, I went library and have been learning JavaScript... it’s not as scary as it looks once you understand the basics, best advice I’ve had in ages 😀

**Comment by rondlite** [comment]
dont have the latest release so guess been in there for a while.. no need for counties that should be automatic

**Comment by Undefined-Variables** [comment]
Yeah I have counties but US states are no use here so I changed the list of states for UK counties like Kent, Berkshire, Aberdeenshire etc, much more useful. I have still got a bit to do but getting there. 

I have changed to much to update it but will have a look at the changes and do them manually if they will benefit my needs!

**Comment by rondlite** [comment]
Yes, dunno where those came from, but the geonames api included in ph7 has any place in the world, so Kent etc are there too..

**Comment by polynamaude** [comment]
@Undefined-Variables I don't know if you speak the pre or post-brexit English language but like @rondlite  said, the Geonames API already include nearly every place in the world where a human has put a feet. So there no such thing as filling out list or whatever you are trying to do. It's all meant to be automatic. In the same way that Google can make a link between your IP and the Country you live in. Based on this, the auto-complete is set to show you cities.

I don't know (and don't want to know) what you are trying here but Ron asked a question that wasn't meant for you. And again, you go off-topic. Is this so hard to understand ?

What he did ask was if the plugins cited in the issue do the same as the auto-complete based on Geonames API. And no, you don't have this answer.

No the question mark is not meant to be a question for you, it's what we call a figure of speech.

**Comment by Undefined-Variables** [comment]
Yeah! I understand how it's supposed to work but when you live in the UK and it still asks you which state in the United States you live in then its not really automatic is it? I only asked as I wanted to know if there was something new in the last release. 

Do you want me to open a new ticket because I wish to comment upon something said in this one? 

Now I will really go of topic! The webcam chat is not done, correct? Its just the meta tags have webcam chat in them but there is no code to support chatting on webcams at all? I have tried a couple of options but the strain webcam chatting puts on the server is really noticeable and that is with one chat happening... Think I am going to have to keep on experimenting. May have to set up another server just to handle webcam chat! 

P. S. That was off topic... Do you see the difference? Now should I open more issues...? Only joking! 
I might just throw an SMF forum up, so I can talk freely about this software without all the stress that you bring to the table! Anyway I don't need to bother asking for help here anymore, I have solved every issue I've ever had on my own but do like to keep an eye on what happens with the software.

That means I might make a comment now and then! Freedom of speech mate :) Beautiful thing!

**Comment by pH-7** [comment]
We might have to use a different library here. Or perhaps, some kind of fallbacks with different GEO providers.

**Comment by pH-7** [comment]
Thanks for the suggestion.

I reviewed this during backlog cleanup. This request is valid but outside the current 18.x delivery scope, so I’m closing it as not planned for now to keep the tracker focused on active work.

If you want to push this forward, please open a new proposal with a minimal implementation plan (scope, DB/API impact, migration notes, and backward-compatibility expectations).

Take care and thank you for using pH7 Social Dating Builder,

[Pierre-Henry](https://github.com/pH-7)
